src/styles/_shared.scss

```diff
@@ -79,5 +79,12 @@ $colors: (
   a {
     color: white;
     font-size: 16px;
+    @media screen and (min-width: 1440px) {
```

Comment:
create scss variable for screen size, see $md

Reply:
yeah, $lg might be good enough in _variable.scss otherwise just call it something like $larger

Reply:
could you also break this out into a seperate scss file? see app.scss and just add a new file in styles folder following the same convention

Reply:
`.seo-list` is a shared class between multiple files and I believe that's why it's there from the beginning, I can split it into it's own file, but keep them as they are now will reduce redundancy. What do you think?

Reply:
ah right, i guess theres not much individual styling so probably some kind of scss mixin would work or just leaving it the way it is. i dont feel strongly enough to make you change it again so ill just merge it